dev-docs/publisher-api-reference/bidderSettings.md

```diff
@@ -344,4 +345,30 @@ pbjs.bidderSettings = {
 
 In the above example, if PubMatic were to return the "groupm" bidder code then the bidCpmAdjustment function under `pubmatic` would be used instead of what is available under `standard`.
 
+##### 2.11. endpointCompression
+
+{: .alert.alert-warning :}
+Note: The bidder's endpoint must already support receiving gzip compressed bid requests (otherwise this feature will not work and most likely throw an error with the bidder's endpoint).
+
+If this setting is enabled (`true`), Prebid.js will attempt to gzip-compress bid requests sent to bidder endpoints when possible.
+
+Compression is performed client-side using the browser's [CompressionStream](https://developer.mozilla.org/en-US/docs/Web/API/CompressionStream) API, if available. If CompressionStream is not supported by the user's browser, the bid request will be sent uncompressed.
+
+This setting can be enabled either:
+
+* globally for all bidders (under the `standard` section), or
+* specifically for one or more bidders.
+
+The default value is `false`, meaning requests are not compressed unless explicitly configured by the publisher.
+
+**Example enabling endpoint compression for a specific bidder (PubMatic):**
+
+```javascript
+pbjs.bidderSettings = {
+  pubmatic: {
+    endpointCompression: true
+  }
+};
+```
```

Comment:
```suggestion
**Example enabling endpoint compression for a specific bidder:**

```javascript
pbjs.bidderSettings = {
  bidderA: {
    endpointCompression: true
  }
};
```
```

Reply:
We try to avoid specific bidder names in the docs

Reply:
Thanks @muuki88!  The requirements on https://github.com/prebid/Prebid.js/pull/13033 have actually changed.  Instead of allowing control to enable/disable endpoint compression on the bidderSettings level, we have most this flag to the request.options level.  I have updated this doc PR to reflect this new change.  Could you review once more when you have a chance?